systemd/org.jfhbrook.plusdeck.conf

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?> <!-- -*- XML -*- -->
+
+<!DOCTYPE busconfig PUBLIC
+ "-//freedesktop//DTD D-BUS Bus Configuration 1.0//EN"
+ "http://www.freedesktop.org/standards/dbus/1.0/busconfig.dtd">
+<busconfig>
+  <policy user="root">
+    <allow own="org.jfhbrook.plusdeck"/>
+  </policy>
+
+  <policy context="default">
+    <allow send_destination="org.freedesktop.plusdeck"/>
```

Comment:
I think this just needs `group=plusdeck` added to it, plus my user being added to the group. Not sure, I'll have to test it out.

Reply:
I was able to, seemingly, get this part working.